examples/bootstrapping-railing/railing.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p-peerstore"
+	"github.com/ipfs/go-ipfs-addr"
```

Comment:
It is prefered to use named import (even the same name as pkg name) if the pkg name is different than last segment of the path.